trace/trace_go16.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build go1.6
+
+package trace
+
+import (
+	"encoding/binary"
+)
+
+// newTraceIDLocked returns a non-zero TraceID from a randomly-chosen sequence.
+// mu should be held while this function is called.
+func newTraceIDLocked() TraceID {
```

Comment:
I'd not repeat newTraceIDLocked in both files. Instead provide a randUint64 for 1.6 and not 1.6.

Reply:
Roger that, thanks for the suggestion.